src/components/profile/ProfileHeader.js

```diff
@@ -0,0 +1,42 @@
+import "./ProfileHeader.css";
+import { Link } from "react-router-dom";
+
+const ProfileHeader = () => {
+  return (
+    <>
+      <div className="profile">
+        <div className="profile-head">
+          <div className="profile-image-container">
+            <img
+              className="profile-image"
+              src="https://images.unsplash.com/photo-1535713875002-d1d0cf377fde?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MXx8YXZhdGFyfGVufDB8MnwwfHw%3D&auto=format&fit=crop&w=500&q=60"
+              alt="profile pics"
+            />
+            <span className="profile-name">
+              <h4 className="profile-name-head">Alex Rohr</h4>
+              <h6 className="profile-name-follow">@alexrohr</h6>
```

Comment:
You shouldn;t jump from `h4` to `h6`

Reply:
This is to make the username bolder for the hierarchy, I could use my className to achieve this instead.

Reply:
Yes you can. Do not use h-tags to avoid adding styles, that's not their purpose